modules/snooker/src/test/ts/atomic/model/TransitionsTest.ts

```diff
@@ -13,7 +13,7 @@ UnitTest.test('TableCounterTest', function () {
   const check = function (expected: Structs.RowDetails[], input: Structs.RowCells[]) {
     const actual = Transitions.toDetails(input, Fun.tripleEquals);
     const cleaner = function (obj: Structs.RowDetails[]) {
-      return Arr.map(obj, function (row) {
+      return Arr.map(obj, (row) => {
```

Comment:
Out of scope for this PR and we disabled that rule, so it's no longer required.

Reply:
I know we don't generally lint for "simplify arrow functions" because some of them are too complex and read better with braces in them. I suggested this one because the body was simple enough that it would probably read better on one line, and I knew that the linter wouldn't be able to fix it.

Reply:
There are far too many cases here to be manually trying to determine if these should or should not use a single line arrow function. If we do want that, we should consider implementing a custom linter rule.

Reply:
I should add all of these were changed using the rules fixer logic, so we don't want to have to trawl through and find every case as that's going to take too long.